dozer-cli/src/cli/helper.rs

```diff
@@ -116,9 +124,11 @@ fn parse_config(config_template: &str) -> Result<Config, CliError> {
     let config: Config = serde_yaml::from_str(&config_str)
         .map_err(|e: serde_yaml::Error| CliError::FailedToParseYaml(Box::new(e)))?;
 
-    // Create home_dir if not exists.
-    fs::create_dir_all(&config.home_dir)
-        .map_err(|e| CliError::FileSystem(config.home_dir.clone().into(), e))?;
+    if prepare_environment_fs {
```

Comment:
I think this block can be removed. `dozer build` will create the directory anyway.